state/state_test.go

```diff
@@ -78,9 +78,9 @@ func TestABCIResponsesSaveLoad(t *testing.T) {
 	// build mock responses
 	block := makeBlock(2, state)
 	abciResponses := NewABCIResponses(block)
-	abciResponses.DeliverTx[0] = &abci.ResponseDeliverTx{Data: []byte("foo")}
-	abciResponses.DeliverTx[1] = &abci.ResponseDeliverTx{Data: []byte("bar"), Log: "ok"}
-	abciResponses.EndBlock = abci.ResponseEndBlock{Diffs: []*abci.Validator{
+	abciResponses.DeliverTx[0] = &abci.ResponseDeliverTx{Data: []byte("foo"), Tags: []*abci.KVPair{}}
```

Comment:
what if we do not simply include Tags field?

Reply:
I think it should be ok to be left empty. isnt it ?

Reply:
it is

Reply:
hmmm.. this breaks TestABCIResponsesSaveLoad

```
--- FAIL: TestABCIResponsesSaveLoad (0.00s)
        Error Trace:    state_test.go:93
        Error:          Not equal:
                        expected: &types.ResponseDeliverTx{Code:0, Data:[]uint8{0x66, 0x6f, 0x6f}, Log:"", Tags:[]*types.KVPair(nil)}
                        actual: &types.ResponseDeliverTx{Code:0, Data:[]uint8{0x66, 0x6f, 0x6f}, Log:"", Tags:[]*types.KVPair{}}
```